### ISSUE TITLE
Error in running the example notebook (AReM_TF)

### ISSUE THREAD

**Issue by aminnayebi** [feedzai/timeshap#10]
Nice work!
I have been trying to run one of the tutorial notebooks in the repository (i.e., AReM_TF), but I faced an error. The notebook chunk that produces the error is:
```
from timeshap.explainer import local_report, local_pruning

pruning_dict = {'tol': 0.025}
event_dict = {'rs': 42, 'nsamples': 320}
feature_dict = {'rs': 42, 'nsamples': 320, 'feature_names': model_features, 'plot_features': plot_feats}
cell_dict = {'rs': 42, 'nsamples': 320, 'top_x_feats': 2, 'top_x_events': 2}
local_report(f, pos_x_data, pruning_dict, event_dict, feature_dict,cell_dict=cell_dict, entity_uuid=positive_sequence_id, entity_col='all_id', baseline=average_event)
```
and the produced error is as follows:
<img width="853" alt="image" src="https://user-images.githubusercontent.com/39872170/162652208-c9da5ced-d5a6-44fd-b4bd-815e84647ce8.png">
<img width="856" alt="image" src="https://user-images.githubusercontent.com/39872170/162652261-49c0ad51-9a83-4d45-8ced-9844c6df8004.png">

It would be great if you could help me with this error.

**Comment by AndreFCruz** [comment]
The input dimensions correspond to (nsamples; seq_len; features), so it seems there's an input sequence with length 0.

Could you give a bit more detail on the error? e.g., your versions for timeshap, tensorflow, and python.
I just checked and it runs fine on my env with `timeshap=0.1.1`, `tensorflow=2.8.0`, and `python=3.7`.

Also, I noticed you're using `nsamples=320` instead of the default 32000, this shouldn't be the root of the issue but it will lead to high variability in the results (or you could select a more aggressive pruning threshold to compensate).

**Comment by AndreFCruz** [comment]
Could you provide the shapes and content of the following variables?
`pos_x_data`
`average_event`

And the shapes of these two as well.
`avg_score_over_len`
`pos_x_pd`

Thanks!

**Comment by aminnayebi** [comment]
Sorry for the late response!
Here are the shape and content of variables that you asked for. It is worth to note that the `avg_score_over_len` is nan for all its key values:
<img width="905" alt="image" src="https://user-images.githubusercontent.com/39872170/164176472-6c08c352-0e02-476d-8d3b-a7c93f42dc9e.png">
<img width="905" alt="image" src="https://user-images.githubusercontent.com/39872170/164176521-ab147b0d-0848-4981-9b8f-f9dc256ec06d.png">
<img width="905" alt="image" src="https://user-images.githubusercontent.com/39872170/164176688-cb96d430-0a3b-4bb2-8406-c0ceef1eacd6.png">

**Comment by JoaoPBSousa** [comment]
Hello @aminnayebi ,

I just ran the AReM_TF notebook from scratch and everything seems to be working on our side. I think the main issue is the `average_event` variable. On our side this variable has value and shape:

![image](https://user-images.githubusercontent.com/36603746/164280893-5e141452-ef8a-4a69-b161-fd466dd034e7.png)

We obtained this variable using the original code: `average_event = calc_avg_event(d_train_normalized, numerical_feats=model_features, categorical_feats=[])`

I would suggest you to pull the notebook from the repo again, and run it without changing anything. This will help us diagnose if the problem is from either the code, the device, or the data.

**Comment by aminnayebi** [comment]
I pulled the repo again and ran the notebook without changing anything. It gave the error again. I could find out what the problem is. The function `calc_avq_event()` is not performing the same for us. The line Probably the version of `pandas` is effective here. My version is `pandas==1.1.3`. Line 45 in the `util.py` file provides an empty data frame with 0 columns and as many rows as the original `data` variable (since there is no list of categorical features in this example).
I changed the `calc_avq_event()` function in a way that only returned the `numerical` variable and it fixed the error.
